docs/pages/guides/authentication.mdx

```diff
@@ -1782,6 +1783,73 @@ const { accessToken } = await AuthSession.exchangeCodeAsync(
 </Box>
 {/* End Strava */}
 
+<Box
+  name="Strivacity"
+  createUrl="https://www.strivacity.com/learn-support/developer-hub"
+  image={ASSETS.strivacity}
+>
+
+| Website                     | Provider  | PKCE      | Auto Discovery |
+| --------------------------- | --------- | --------- | -------------- |
+| [Get Your Config][c-strivacity] | OpenID | Supported | Available  |
+
+[c-strivacity]: https://docs.strivacity.com/docs/setting-up-your-instance
```

Comment:
```suggestion
```

Since there's only one time this linked is used, we don't neet to define it like this.